src/Tickets/Emails/Admin/Preview_Data.php

```diff
@@ -17,6 +17,54 @@
  * @package TEC\Tickets\Emails\Admin
  */
 class Preview_Data {
+	/**
+	 * Get default preview data.
+	 *
+	 * @since TBD
+	 *
+	 * @return array The default preview data.
+	 */
+	public static function get_default_preview_data(): array {
+		$current_user = wp_get_current_user();
+		$title        = empty( $current_user->first_name ) ?
+		__( 'Here\'s your ticket!', 'event-tickets' ) :
+		sprintf(
+			// Translators: %s - First name of email recipient.
+			__( 'Here\'s your ticket, %s!', 'event-tickets' ),
+			$current_user->first_name
+		);
```

Comment:
Make into a full conditional.

Reply:
@bordoni I am just moving things from one place to the other. I will change it.